src/silvimetric/resources/metric.py

```diff
@@ -160,84 +212,115 @@ def m_cumean(data):
 # statement with derived works)
 
 # L1 is same as mean...compute using np.mean for speed
-def m_l1(data):
+def m_l1(data, htthreshold, coverthreshold):
+    if len(data) < 4:
+        return -9999.0
+
     return np.mean(data)
 
-def m_l2(data):
+def m_l2(data, htthreshold, coverthreshold):
+    if len(data) < 4:
+        return -9999.0
+
     l = lmom4(data)
     return l[1]
 
-def m_l3(data):
+def m_l3(data, htthreshold, coverthreshold):
+    if len(data) < 4:
+        return -9999.0
+
     l = lmom4(data)
     return l[2]
 
-def m_l4(data):
+def m_l4(data, htthreshold, coverthreshold):
```

Comment:
We really need some kind of docstring on all of these methods. We want to be able to reflect the docstring back into the Metric definition that's stored in the database so that it makes it out for anyone doing queries.

Reply:
Do we do this as a comment or something more complicated?

Reply:
> Do we do this as a comment

yep, we can just use normal Python docstrings, which are comments after the function definition.